frontend/package.json

```diff
@@ -44,5 +45,8 @@
       "last 1 safari version"
     ]
   },
-  "proxy": "http://localhost:8080"
+  "proxy": "http://localhost:8080",
+  "devDependencies": {
+    "@types/react-datepicker": "^4.11.2"
```

Comment:
Is this packages only needed during development? If the packages is required for your application to run correctly in a production environment, you should include as dependency instead of devdependency.